src/action/common/place_nix_configuration.rs

```diff
@@ -57,6 +57,9 @@ impl PlaceNixConfiguration {
             "extra-nix-path".to_string(),
             "nixpkgs=flake:nixpkgs".to_string(),
         );
+
+        // Auto-allocate uids is broken on Mac. Tools like `whoami` don't work.
```

Comment:
May be a nice idea to link to the issue I filed about this:
```suggestion
        // Auto-allocate uids is broken on Mac. Tools like `whoami` don't work.
        // e.g. https://github.com/NixOS/nix/issues/8444
```